src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -36,6 +36,12 @@ public interface IBlockRepository : IDisposable
         /// <param name="hash">The block hash.</param>
         Task<Block> GetAsync(uint256 hash);
 
+        /// <summary>
+        /// Get the blocks from the database by using block hashes.
+        /// </summary>
+        /// <param name="hash">The block hash.</param>
+        Task<List<Block>> GetBlocksAsync(List<uint256> hashes);
```

Comment:
Please add specifications of the return value and make note that the blocks are returned in the same order as the hashes on input and that if a block is not found a null value is returned at its spot.

Reply:
Done.